---
layout: protocols
title: Commonly used E. coli genotypes
author: Shaohe Wang
author_url: https://scholar.google.com/citations?user=R_-kJV4AAAAJ&hl=en
date: 2020-07-16
---

This [Addgene blog](https://blog.addgene.org/plasmids-101-common-lab-e-coli-strains) is a great read for _E. coli_ strains. You can refer to [OpenWetWare](https://openwetware.org/wiki/E._coli_genotypes) and [Addgene](https://blog.addgene.org/plasmids-101-common-lab-e-coli-strains) for a more complete list of _E. coli_ strains and genotype meaning. Here we list the ones we commonly use.

### Commonly used _E. coli_ strains

* __DH5α__: _F<sup>-</sup> endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG purB20 φ80d[lacZΔM15] Δ(lacZYA-argF)U169, hsdR17(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup>) λ<sup>-</sup>_
* __Endura electrocompetent cells (same as HB101)__: _F<sup>-</sup> mcrB mrr hsdS20(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>) recA13 leuB6 ara-14 proA2 lacY1 galK2 xyl-5 mtl-1 rpsL20(Sm<sup>R</sup>) glnV44 λ<sup>-</sup>_
* __NEB Stable__: _F'[proA<sup>+</sup>B<sup>+</sup> lacI<sup>q</sup> lacZΔM15] zzf::Tn10 (Tet<sup>R</sup>) ∆(ara-leu)7697 araD139 fhuA ∆lacX74 galK16 galE15 e14- φ80d[lacZΔM15] recA1 relA1 endA1 nupG rpsL (Str<sup>R</sup>) rph spoT1 ∆(mrr-hsdRMS-mcrBC)_
* __NEB Turbo__: _F'[proA<sup>+</sup>B<sup>+</sup> lacI<sup>q</sup> lacZΔM15] fhuA2 ∆(lac-proAB) glnV44 galK16 galE15  R(zgb-210::Tn10)Tet<sup>S</sup> endA1 thi-1 ∆(hsdS-mcrB)5 r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>-</sup>_
* __BJ5183__ (Hanahan D, 1983; Note: hsdR may be actually hsdS): _F<sup>-</sup> endA sbcB<sup>-</sup> recBC<sup>-</sup> galK met<sup>-</sup> str<sup>R</sup> thi-1, bloT hsdR(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>-</sup>) λ<sup>-</sup>_
* __BL21__: _[E. coli strain B] F<sup>-</sup> ompT gal dcm lon hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>) [malB<sup>+</sup>]<sub>K-12</sub>(λ<sup>S</sup>)_
* __BL21(DE3)__: _[E. coli strain B] F<sup>-</sup> ompT gal dcm lon hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>) λ(DE3 [lacI<sup>q</sup> lacUV5-T7p07 ind1 sam7 nin5]) [malB<sup>+</sup>]<sub>K-12</sub>(λ<sup>S</sup>)_
* __BL21(DE3) pLysS__: _[E. coli strain B] F<sup>-</sup> ompT gal dcm lon hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>) λ(DE3 [lacI<sup>q</sup> lacUV5-T7p07 ind1 sam7 nin5]) [malB<sup>+</sup>]<sub>K-12</sub>(λ<sup>S</sup>) pLysS\[T7p20 ori<sub>p15A</sub>\](Cm<sup>R</sup>)_
* __BL21(DE3) pLysE__: _[E. coli strain B] F<sup>-</sup> ompT gal dcm lon hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>) λ(DE3 [lacI<sup>q</sup> lacUV5-T7p07 ind1 sam7 nin5]) [malB<sup>+</sup>]<sub>K-12</sub>(λ<sup>S</sup>) pLysE\[T7p20 ori<sub>p15A</sub>\](Cm<sup>R</sup>)_
* __Rosetta™(DE3)pLysS__: _[E. coli strain B] F<sup>-</sup> ompT gal dcm lon? hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>) λ(DE3 [lacI<sup>q</sup> lacUV5-T7p07 ind1 sam7 nin5]) [malB<sup>+</sup>]<sub>K-12</sub>(λ<sup>S</sup>) pLysSRARE\[T7p20 ileX argU thrU tyrU glyT thrT argW metT leuW proL ori<sub>p15A</sub>\](Cm<sup>R</sup>)_


### _E. coli_ genotype meaning

* __ara-14__: Cannot metabolize arabinose.
* __DE3__: A λ prophage (λ lysogen) carrying the T7 RNA polymerase gene under control of the lacUV5 promoter and lacI<sup>q</sup>.
* __deoR__: Constitutive expression of genes for deoxyribose synthesis. Allows uptake of large plasmids.
* __Δ(lacZYA-argF)U169__: Deletion of a chromosomal region including the lac operon and argF in the allele U169 Enhances resistance to hydrogen peroxide ([reference](https://www.thermofisher.com/us/en/home/life-science/cloning/cloning-learning-center/invitrogen-school-of-molecular-biology/molecular-cloning/transformation/competent-cell-genotypes-genetic-markers.html)).
* __endA1__: Endonuclease I mutation that eliminates non-specific digestion by Endonuclease I.
* __F<sup>-</sup>__: Does not carry the F plasmid. The F plasmid allows E. coli to mate through conjugation.
* __F'[list-of-host-genes]__: Carries an F plasmid that has host genes from a previous recombination event. Same as F<sup>+</sup> cells, it can also mate with F<sup>-</sup> through conjugation.
* __fhuA__ (formerly tonA): Ferric hydroxamate uptake, iron uptake receptor mutation. Resistance to phage T1/T5.
* __galK, galK16, galK2__: Mutation in the galactokinase gene. Cannot metabolize galactose and are resistant to 2-deoxygalactose. galK16 is an IS2 insertion ~170bp downstream of the galK start codon. galK2 is an ochre mutation (G->T at base pair 571; GAA->UAA).
* __glnV44__ (supE44): Suppression of the amber (UAG) stop codon by glutamine insertion.
* __gyrA, gyrA96__: DNA gyrase mutation. Confers resistance to nalidixic acid.
* __hsdR17(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup>)__: Unmethylated DNA not degraded, but cells can still methylate DNA.
* __hsdS(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>-</sup>), hsdSB(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>), hsdS20(r<sub>B</sub><sup>-</sup>m<sub>B</sub><sup>-</sup>)__: Unmethylated DNA not degraded AND cell cannot methylate DNA.
* __lacY1__: Mutation in the galactoside permease gene. Blocks lactose utilization.
* __lacZ__: β-galactosidase activity abolished.
* __lacI<sup>q</sup>__: Mutation in the lac repressor gene for constitutive expression. Inhibits transcription from the lac promoter, which can be overcome by IPTG addition. Allows controlled gene expression from promoters that carry the lac operator. The superscript <sup>q</sup> indicates a constitutive mutation.
* __λ<sup>-</sup>__: Deletion of λ lysogen.
* __leuB6__: Requires leucine.
* __[malB<sup>+</sup>]<sub>K-12</sub>(λ<sup>S</sup>)__: E. coli B strain was known to be Mal<sup>-</sup> and λ resistant. In order to work with λ in B strain, Mal<sup>+</sup> and λ sensitive derivatives were isolated after P1 transduction from W3110. See Studier et al., 2009, JMB for more details.
* __mcrA__: Mutation eliminating restriction of DNA methylated at the sequence C<sup>m</sup>CGG (possibly <sup>m</sup>CG). Carried on the e14 prophage (q.v.)
* __mcrB__: Mutation eliminating restriction of DNA methylated at the sequence R<sup>m</sup>C
* __metB, metC__: Requires methionine.
* __mrr__: Mutation eliminating restriction of DNA methylated at the sequence C<sup>m</sup>AG or G<sup>m</sup>AC
* __mtl-1__: The mannitol utilization is blocked.
* __nupG__: The gene encodes a broad-specificity transporter of purine and pyrimidine nucleosides. Driven by a proton motive force. Can transport uridine, adenosine, inosine, guanosine, thymidine and cytidine. [OpenWetWare](https://openwetware.org/wiki/E._coli_genotypes) says this is the same as deoR.
* __φ80__: Cell carries the lambdoid prophage φ80. A defective version of this phage (φ80d[lacZΔM15_) carrying lacZM15 deletion (as well as wild-type lacI, lacYA, and flanking sequences) is present in some strains. The φ80 attachment site is just adjacent to tonB.
* __proA2__: Requires proline.
* __purB20__: The purB encodes a protein that is involved in step 2 of the sub-pathway that synthesizes AMP from IMP.
* __pLysE__: The pLysE plasmid encodes T7 phage lysozyme, an inhibitor for T7 polymerase which almost eliminates expression from transformed T7 promoter containing plasmids when not induced. pLysE produces higher amounts of T7 lysozyme than pLysS to further reduce basal level expression of recombinant genes by inhibiting basal levels of T7 RNA polymerase. This makes BL21(DE3)[pLysE] more suitable for expressing toxic genes.
* __pLysS__: The pLysS plasmid encodes T7 phage lysozyme, an inhibitor for T7 polymerase which almost eliminates expression from transformed T7 promoter containing plasmids when not induced.
* __pLysSRARE__: In addition to T7 phage lysozyme, pLysSRARE also contains tRNA genes argU, argW, ileX, glyT, leuW, proL, metT, thrT, tyrU, and thrU. The rare codons AGG, AGA, AUA, CUA, CCC, and GGA are supplemented.
* __recA, recA1, recA13__: Mutation in a DNA-dependent ATPase that is essential for recombination and general DNA repair. It reduces plasmid recombination and increases plasmid stability.
* __relA, relA1__: Relaxed phenotype. Mutation eliminating stringent factor	and thus allows RNA synthesis in absence of protein synthesis.
* __rpsL20(Sm<sup>R</sup>)__: Mutation in ribosomal protein S12 conveying streptomycin resistance.
* __thi-1__: Mutation in thiamine metabolism. Requires exogenous thiamine for growth.
* __xyl-5__: Mutation in a xylose metabolism gene. Blocks xylose utilization.
